osf/metrics/reporters/spam_count.py

```diff
@@ -33,6 +38,40 @@ def report(self, report_yearmonth):
                 created__lt=next_month,
                 node__type='osf.node',
             ).count(),
+            node_flagged_reversed=Node.objects.filter(
+                logs__action=NodeLog.FLAG_SPAM
+            ).filter(
+                logs__action=NodeLog.CONFIRM_HAM,
+                logs__created__gt=target_month,
+                logs__created__lt=next_month,
+            ).distinct().count(),
+            node_flagged_reversed_akismet=Node.objects.filter(
+                logs__action=NodeLog.FLAG_SPAM
+            ).filter(
+                logs__action=NodeLog.CONFIRM_HAM,
+                logs__created__gt=target_month,
+                logs__created__lt=next_month,
+            ).filter(
+                spam_data__who_flagged='akismet'
+            ).distinct().count(),
+            node_flagged_reversed_oopspam=Node.objects.filter(
+                logs__action=NodeLog.FLAG_SPAM
+            ).filter(
+                logs__action=NodeLog.CONFIRM_HAM,
+                logs__created__gt=target_month,
+                logs__created__lt=next_month,
+            ).filter(
+                spam_data__who_flagged='oopspam'
+            ).distinct().count(),
+            node_flagged_reversed_both=Node.objects.filter(
+                logs__action=NodeLog.FLAG_SPAM
+            ).filter(
+                logs__action=NodeLog.CONFIRM_HAM,
+                logs__created__gt=target_month,
+                logs__created__lt=next_month,
+            ).filter(
+                spam_data__who_flagged='both'
```

Comment:
consider consolidating the `both` into the akismet and the oopspam? Double check with product on what is best for them on this